src/_data/sidenav.yml

```
@@ -331,6 +331,8 @@
       permalink: /data-and-backend/firebase
     - title: Google APIs
       permalink: /data-and-backend/google-apis
+    - title: Concurrency and Isolates
+      permalink: data-and-backend/isolates
```

To get the stage workflow running:

```suggestion
      permalink: /data-and-backend/isolates
```